1_Installation_Files (v1.5.5)/includes/classes/bmz_image_handler.class.php

```diff
@@ -478,6 +478,11 @@ protected function getCacheName($data, $ext='')
     public function calculate_size($pref_width, $pref_height = '')
     {
         if (file_exists($this->filename)) {
+            $image_info = @getimagesize($this->filename);//silence multiple default error messages
+            if ($image_info === false){
+                error_log("Image Handler, calculate_size($this->filename) returned FALSE: image is corrupt");
```

Comment:
Use `trigger_error`, with an `E_USER_NOTICE` severity, instead of `error_log`.

Reply:
ok, but there is more to this in the rabbit hole of zen_image, IH and instant search that I still trying to resolve, not before Friday! :persevere: